tools/pipelines/build-docs-website-deploy.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# build-docs-website-deploy pipeline
+
+name: $(Build.BuildId)
+
+schedules:
+  - cron: "0 */4 * * *"
+    displayName: Website deploy with 4 hour increments
+    branches:
+      include:
+        - main
+    always: true
+
+trigger:   
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - docs
+    - "**/api-report"
```

Comment:
Let's add a brief comment to this line explaining why it's relevant. It may not be immediately obvious.

Reply:
Actually, I'm not sure this gets us what we want. If this pipeline runs on changes to main under these directories, it will run at the same time as the corresponding `build - foo` pipelines, which means the artifacts we're waiting for might not be ready yet. I suspect it will put us in a consistently off-by-1 state, where we end up publishing for everything except the newest artifacts.

Reply:
I wonder if what we really want is for `build-docs` to run off of these triggers (on the appropriate branch), and for this pipeline to trigger on the completion of `build-docs` (from any branch) and always run on main?

Reply:
Makes sense! I moved the docs file trigger to build-docs and added a new trigger to the deploy pipeline which looks for main branch build-docs runs